akka-http/src/main/scala/akka/http/scaladsl/server/ExceptionHandler.scala

```diff
@@ -40,13 +40,11 @@ object ExceptionHandler {
   def default(settings: RoutingSettings): ExceptionHandler =
     apply(knownToBeSealed = true) {
       case IllegalRequestException(info, status) ⇒ ctx ⇒ {
-        ctx.log.warning(
-          "Illegal request {}\n\t{}\n\tCompleting with '{}' response",
-          ctx.request, info.formatPretty, status)
+        ctx.log.warning("Illegal request. Completing with {} response.", status)
         ctx.complete((status, info.format(settings.verboseErrorMessages)))
       }
       case NonFatal(e) ⇒ ctx ⇒ {
-        ctx.log.error(e, "Error during processing of request {}", ctx.request)
+        ctx.log.error(e, "Error during processing of request.", ctx.request)
```

Comment:
`ctx.request` seems unused.  Either remove or change to something else and add back `{}`.